main/inc/global-min.inc.php

```diff
@@ -0,0 +1,303 @@
+<?php
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4488007)